### DISCUSSION TITLE
Trouble getting versioning to work

### DISCUSSION THREAD

**Discussion by Warkanlock** [projen/projen#2389] [Q&A] [answered]
I have a hard time understanding how versioning work inside Projen.

I'm trying to understand how Projen checks the latest version, but I couldn't simply follow the documentation.

The following text simply does not match what are I'm seeing
``For major versions 1 and above, if a release includes fix commits only, it will increase the patch version. If a release consists of any feat saves, then the new version will be a minor version.``

I'm creating commits using ``feat`` and even breaking changes with ``release, `` but nothing has happened. The only thing is that the first commit with the breaking change version was a bump to 1

Any insights on how to do this? 
Thanks

**Comment by Warkanlock** [comment, marked as answer]
I figured out that tag default prefix is ``v*`` so I created a PR explaining it on the documentation:

https://github.com/projen/projen/pull/2390

**Reply by mrgrain** [reply]
Thanks for updating the docs! Was that the main bit of information you were missing? Do you still have open questions?

**Reply by Warkanlock** [reply]
Exactly! I was using a default tag but without the ``v* ``
